android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -1429,7 +1428,7 @@ private void videoLoaded() {
             ArrayList<Track> audioTracks = getAudioTrackInfo();
             ArrayList<Track> textTracks  = getTextTrackInfo();
 
-            if (this.contentStartTime != -1L) {
+            if (source.getContentStartTime() != -1L) {
```

Comment:
```suggestion
     source.getContentStartTime() != -1
```
Because contentStartTime is Int.

Reply:
yes make sense ! Done